DXRMapFixups/DeusEx/Classes/DXRFixupM06.uc

```diff
@@ -303,14 +316,20 @@ function PreFirstEntryMapFixes()
                 }
             }
 
+        } else {
+            //behind Maggie's DispalyCase (sic), there is a Trigger to open it
+            //That trigger gets hit when an OrdersTrigger in the same spot gets replaced by DXRReplaceActors
```

Comment:
should this just always happen? setting to pawnproximity can't ever hurt?

Reply:
The equivalent of this probably does exist in Vanilla Maps as well, and same thing would happen in Revision in that case, so probably?

Unless you mean *all* Triggers?  In which case it seems a bit overkill (maybe)

Reply:
I meant just moving this block out of the else block, it doesn't happen in the if(VanillaMaps) block because the trigger doesn't get replaced there, but this might help for GMDX

Reply:
k